docs/zero_to_hero_guide/quickstart.md

```diff
@@ -0,0 +1,185 @@
+
+
+
+## Setting up
+
+### 1. Prerequisite
+
+Ensure you have the following installed on your system:
+
+- **Conda**: A package, dependency, and environment management tool.
+
+
+### 2. Installation
+The `llama` CLI tool helps you manage the Llama Stack toolchain and agent systems.
+
+
+```bash
+pip install llama-stack
+```
+
+After installation, the `llama` command should be available in your PATH.
+
+### 3. Download Llama Models
+
+Download the necessary Llama model checkpoints using the `llama` CLI:
+
+```bash
+llama download --model-id Llama3.2-3B-Instruct
+```
+
+Follow the CLI prompts to complete the download. You may need to accept a license agreement. Obtain an instant license [here](https://www.llama.com/llama-downloads/).
+
+---
+
+## Build, Configure, and Run Llama Stack
+
+### 1. Build the Llama Stack Distribution
+
+We will default to building the `meta-reference-gpu` distribution due to its optimized configuration tailored for inference tasks that utilize local GPU capabilities effectively. If you have limited GPU resources, prefer using a cloud-based instance or plan to run on a CPU, you can explore other distribution options [here](https://llama-stack.readthedocs.io/en/latest/getting_started/index.html#decide-your-inference-provider).
+
+```bash
+llama stack build --template meta-reference-gpu --image-type conda
+```
+
+
+### 2. Run the Llama Stack Distribution
+> Launching a distribution initializes and configures the necessary APIs and Providers, enabling seamless interaction with the underlying model.
+
+Start the server with the configured stack:
+
+```bash
+cd llama-stack/distributions/meta-reference-gpu
+llama stack run ./run.yaml
+```
+
+The server will start and listen on `http://localhost:5000` by default.
+
+---
+
+## Testing with `curl`
+
+After setting up the server, verify it's working by sending a `POST` request using `curl`:
+
+```bash
+curl http://localhost:5000/inference/chat_completion \
+-H "Content-Type: application/json" \
+-d '{
+    "model": "Llama3.2-3B-Instruct",
+    "messages": [
+        {"role": "system", "content": "You are a helpful assistant."},
+        {"role": "user", "content": "Write me a 2-sentence poem about the moon"}
+    ],
+    "sampling_params": {"temperature": 0.7, "seed": 42, "max_tokens": 512}
+}'
+```
+
+**Expected Output:**
+```json
+{
+  "completion_message": {
+    "role": "assistant",
+    "content": "The moon glows softly in the midnight sky,\nA beacon of wonder, as it catches the eye.",
+    "stop_reason": "out_of_tokens",
+    "tool_calls": []
+  },
+  "logprobs": null
+}
+```
+
+---
+
+## Testing with Python
+
+You can also interact with the Llama Stack server using a simple Python script. Below is an example:
+
+### 1. Install Required Python Packages
+The `llama-stack-client` library offers a robust and efficient python methods for interacting with the Llama Stack server.
+
+```bash
+pip install llama-stack-client
+```
+
+### 3. Create Python Script (`test_llama_stack.py`)
```

Comment:
numbering is wrong, but also can we just inline all these steps into paragraphs and not make them into sections each of which is just 2 sentences? Just `## Testing with Python` is enough followed by the content directly inline separated nicely into code blocks and paras. No sub-sub-sections or numbering needed.

Reply:
hi @ashwinb, changed the numbering, thanks for the suggestion. I’m happy to switch to the inline style. However, I believe numbering makes the steps easier to follow